plugins/builtin/apm/nomad/plugin/metrics.go

```diff
@@ -27,7 +29,9 @@ const (
 )
 
 // Query satisfies the Query function on the apm.APM interface.
-func (a *APMPlugin) Query(q string) (float64, error) {
+func (a *APMPlugin) Query(q string, _ sdk.TimeRange) (sdk.TimestampedMetrics, error) {
+	// The Nomad Metrics API doesn't provide historical data, so time range
+	// for the query is not used.
```

Comment:
Curiosity question; is there is a trick I am missing here and some IDE magic; how comes the comment regarding the input parameter is inside the function rather than as the function comment?

Reply:
At first I thought this would be more a developer-focused comment, so I didn't want it to show in the Go docs. 

But looking at it again, it's probably even more relevant to external callers, since this function will always return a single value regardless of the input range.

I will move it outside, thanks for flagging it 👍